projects/optic/src/init.ts

```diff
@@ -63,14 +63,13 @@ export const initCli = async () => {
   registerInit(cli, cliConfig);
   registerDiff(cli, cliConfig);
 
-  const cloudSubcommands = cli
-    .command('cloud')
+  const rulesetSubcommands = cli
+    .command('ruleset')
     .description(
-      'Commands to interact with Optic Cloud. See `optic cloud --help`'
+      'Commands to build your own optic rulesets. See `optic ruleset --help`'
     )
     .addHelpCommand(false);
-  registerCreateGithubContext(cloudSubcommands, true);
-  registerCreateManualContext(cloudSubcommands);
```

Comment:
Removed these two since we're no longer running CI triggered comparisons, and therefore these are not used for anything in the `optic` binary